recipes/openblas/all/conanfile.py

```diff
@@ -50,6 +50,10 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    def build_requirements(self):
+        if self.options.build_lapack:
+            self.requires("gfortran/10.2")
```

Comment:
```suggestion
            self.tool_requires("gfortran/10.2")
```
self.requires should be used only for def requirements()